enterprise/views.py

```diff
@@ -486,6 +487,18 @@ def get_enterprise_course_enrollment_page(self, request, enterprise_uuid, course
             logger.error('Unable to find enterprise customer for UUID: %s', enterprise_uuid)
             raise Http404
 
+        if EnterpriseCourseEnrollment.objects.filter(
+                enterprise_customer_user__enterprise_customer=enterprise_customer,
+                enterprise_customer_user__user_id=request.user.id,
+                course_id=course_id
+        ).exists():
+            # The user is already enrolled in the course through the Enterprise Customer, so redirect to the course
+            # info page.
+            return redirect(urljoin(
+                settings.LMS_ROOT_URL,
+                '/courses/{course_id}'.format(course_id=course_id)
+            ))
```

Comment:
Do you think there's a reasonable way I should write this setting into the enterprise app only (without adding the setting to edx-platform)? It feels really odd adding `LMS_COURSE_URL` to the LMS setting envs, whether using the reverse router to do so or not.